rules-reviewed/jboss-eap/websphere/tests/websphere.windup.test.xml

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0"?>
+<ruletest xmlns="http://windup.jboss.org/v1/xml">
+    <testDataPath>data</testDataPath>
+    <rulePath>../WebSphere.windup.xml</rulePath>
+    <ruleset>
+        <rules>
+            <rule>
```

Comment:
IMO, these should have ids and something tying them back to the original rule that they are testing. If the ids are related, that would be sufficient, IMO.